Source/Core/DolphinWX/LuaScripting.h

```diff
@@ -0,0 +1,101 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <map>
+#include <math.h>
+#include <mutex>
+#include <wx/artprov.h>
+#include <wx/bitmap.h>
+#include <wx/button.h>
+#include <wx/colour.h>
+#include <wx/filedlg.h>
+#include <wx/font.h>
+#include <wx/frame.h>
+#include <wx/gdicmn.h>
+#include <wx/icon.h>
+#include <wx/image.h>
+#include <wx/menu.h>
+#include <wx/settings.h>
+#include <wx/sizer.h>
+#include <wx/stattext.h>
+#include <wx/string.h>
+#include <wx/textctrl.h>
+#include "Core\HW\ProcessorInterface.h"
+#include "Core\State.h"
+#include "InputCommon\GCPadStatus.h"
+
+// Lua includes
+#ifdef __cplusplus

```

Comment:
These can likely be moved to the cpp file, with lua_State, just being forward declared.